### PR TITLE
Revamping a few single panel figures for posters and later

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR adjusts some of the graphics ans visualizations for a few Elastic net model performance figures. Please let me know if you have any questions!

### CODE REVIEW

File: 6.bulk_Morphology_Elastic_Network/2.test_model/figures/regression/PBMC/aggregated_with_nomic/local_variance_r2_legend.png
Comment (jenna-tomkinson) [inline 6.bulk_Morphology_Elastic_Network/2.test_model/figures/regression/PBMC/aggregated_with_nomic/local_variance_r2_legend.png:1]: Question: What does removing the legend from the plot and having it's own PNG help with when generating these plots?

File: 6.bulk_Morphology_Elastic_Network/2.test_model/scripts/3.visualize_regression_multi_output.r [hunk 1]
```diff
@@ -51,42 +41,88 @@ if (!file.exists(dirname(global_prediction_trend_path))) {
     dir.create(dirname(global_prediction_trend_path), recursive = TRUE)
 }
 # plot the data
-global_prediction_trend <- (
+global_prediction_trend_scatter <- (
     ggplot(df, aes(x=actual_value, y=predicted_value, col=shuffle))
-    + geom_point()
-    + geom_smooth(method=lm, se=TRUE,)
+    + geom_point(alpha=0.5, size=0.5)
+    # add geom smooth with each line being a different color
```
Comment (jenna-tomkinson) [inline 6.bulk_Morphology_Elastic_Network/2.test_model/scripts/3.visualize_regression_multi_output.r:47]: I am a bit confused what this code comment references, looks like geom_smooth is deleted.

**Review by jenna-tomkinson — APPROVED** [review]
Great PR! R is a very fun language to try and read. I made a few comments, the main comments come from the figures.

I am excited to see more figures!

File: 6.bulk_Morphology_Elastic_Network/2.test_model/figures/regression/PBMC/aggregated_with_nomic/local_variance_r2_legend.png
Comment (MikeLippincott) [reply 6.bulk_Morphology_Elastic_Network/2.test_model/figures/regression/PBMC/aggregated_with_nomic/local_variance_r2_legend.png:1]: @jenna-tomkinson helps when I want to have full manipulation of the legend after figure generation (i.e. for poster, or slide deck) plus if the figure dimension are 4x6 I know that it is 4X6 for the plot and not the whole figure (the legend skews it...) not crucial though and definitely my one of my preferences.

File: 6.bulk_Morphology_Elastic_Network/2.test_model/scripts/3.visualize_regression_multi_output.r (hunk 1 above)
Comment (MikeLippincott) [reply 6.bulk_Morphology_Elastic_Network/2.test_model/scripts/3.visualize_regression_multi_output.r:47]: I needed to remove that line of comment...

File: 6.bulk_Morphology_Elastic_Network/2.test_model/figures/regression/PBMC/aggregated_with_nomic/global_prediction_trend.png
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply 6.bulk_Morphology_Elastic_Network/2.test_model/figures/regression/PBMC/aggregated_with_nomic/global_prediction_trend.png:1]: I was able to address all except for 4 which I would need to mutate the df.

**Pull request merged**